README.md

```diff
@@ -54,7 +54,7 @@ We welcome your feedback and contributions in any form possible.
   - Already signed up? Head to our discussions at [#openebs-users](https://openebs-community.slack.com/messages/openebs-users/)
 - Want to raise an issue?
   - If it is a generic (or `not really sure`), you can still raise it at [issues](https://github.com/openebs/openebs/issues)
-  - Project specific issues can be raised at individual project level.
+  - Project specific issues can be raised at github.com/openebs/openebs/issues with the individual project tag.
```

Comment:
[issues] (https://github.com/openebs/openebs/issues) with the individual project tag.